experiment/simulation/index.html

```diff
@@ -1,13 +1,226 @@
+                    <span class="br"></span>
+                    <button id="four-reset" onclick="reset(event)" class="btn">RESET</button>
+                    <span class="br"></span>
+                    <button class="btn" id="five-switch"><a href="html/CO1_5-bit.html">5-bit Table</a></button>
+                    <span class="br"></span>
+                </div>
+                <div>
+                    <h3 id="result-heading">Results</h3>
+                    <div class="4-bit-result-table">
+                        <table class="rtable table-bordered" id="result">
+                            <tbody id="anku">
+                                <tr id="rowbinar">
+                                    <td>1</td>
+                                    <th>Binary Rep</th>
+                                    <td class="ans" id="binar"></td>
 
-	<!-- Add JS at the bottom of HTML file --> 
-        <script src="./js/main.js"></script>
+                                </tr>
+                                <tr id="row2s">
+                                    <td>2</td>
+                                    <th>2's Comp</th>
+                                    <td class="ans" id="2s"></td>
+                                </tr>
+                                <tr id="row1s">
+                                    <td>3</td>
+                                    <th>1's Comp</th>
+                                    <td class="ans" id="1s"></td>
+                                </tr>
+                                <tr id="rowunsigned">
+                                    <td>4</td>
+                                    <th>Unsigned</th>
+                                    <td id="unsigned"></td>
+                                </tr>
+                                <tr id="rowsigned">
+                                    <td>5</td>
+                                    <th>Signed</th>
+                                    <td id="signed"></td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-7">
+                <div class="bit-table m-auto pt-5">
+                    <table id="four-bit-table">
+                        <thead>
+                            <tr>
+                                <th scope="col" id="me">#</th>
+                                <th id="ef" scope="col">Binary</th>
+                                <th scope="col">2's Comp</th>
+                                <th scope="col">1's Comp</th>
+                                <th id="ab" scope="col">Unsigned</th>
+                                <th id="cd" scope="col">Signed</th>
+                            </tr>
+                        </thead>
+
+                        <tbody>
+                            <tr id="1" onclick="rowSelection(event,'1')" onmouseover="mouseOver(event)" onmouseout="mouseOut(event)">
```

Comment:
ids should be unique in an html document. If we do not need ids for the td elements, we can remove them. Otherwise, give them unique ids and try to keep them meaningful to increase readability

Reply:
Sir, There is a purpose for making a set of **td** as "1", "2",.. till table values which is to color hovered or selected row where is the whole row is colored all time and class names are different because for highliting result which won't be a whole row in all the cases.